desafio17.sql

```diff
@@ -0,0 +1 @@
+SELECT `id`, supplier_id FROM northwind.purchase_orders WHERE (supplier_id = 1 OR supplier_id = 3 OR supplier_id = 5 OR supplier_id = 7);
```

Comment:
@renanpbraga no desafio 17, também como forma de melhoria de desempenho, outra forma de escrever a query é:

```suggestion
SELECT id, supplier_id FROM northwind.purchase_orders WHERE supplier_id IN (1,3,5,7);
```